packages/components/nfts/NFTs.tsx

```diff
@@ -53,21 +52,38 @@ export const NFTs: React.FC<{
             }}
           />
         )}
-        <GridList<NFT>
-          data={nfts}
-          onEndReached={handleEndReached}
-          keyExtractor={keyExtractor}
-          renderItem={(info, elemWidth) => (
-            <NFTView
-              key={info.item.id}
-              data={info.item}
-              style={{ width: elemWidth }}
-            />
-          )}
-          minElemWidth={250}
-          gap={layout.spacing_x2}
-          noFixedHeight={Platform.OS !== "web"}
-        />
+        {Platform.OS === "web" ? (
+          <GridList<NFT>
+            data={nfts}
+            onEndReached={handleEndReached}
+            keyExtractor={keyExtractor}
+            renderItem={(info, elemWidth) => (
+              <NFTView
+                key={info.item.id}
+                data={info.item}
+                style={{ width: elemWidth }}
+              />
+            )}
+            minElemWidth={250}
+            gap={layout.spacing_x2}
+          />
+        ) : (
+          <FlatList
+            data={nfts}
```

Comment:
Since header and other options are on other components. Just created FlatList for mobile

Reply:
I think this is the same behavior, gridlist is already a flatlist, with noFixedHeight it's like the flatlist you added, did you check network request to see if it does not load everything?

Reply:
Yeah, it's same.
I did check on item length and it was fetching next 100 item at a time.

Reply:
Thanks for confirming